tests/bearlib/aspects/BaseTest.py

```diff
@@ -15,5 +15,7 @@ def test_class(self):
         assert not isinstance(aspectbase, aspectclass)
 
     def test_init_needs_aspectclass(self):
-        with pytest.raises(AttributeError):
+        error_str = 'object has no attribute'
+        with pytest.raises(AttributeError) as exc:
```

Comment:
The issue is about removing dependency on pytest-specific functionality in the tests.